frontend/src/features/Base/components/BaseOverlay/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { useMemo } from 'react'
+
+import { BaseCard } from './BaseCard'
+import { OVERLAY_MARGINS } from './constants'
+import { Layers } from '../../../../domain/entities/layers/constants'
+import { useAppSelector } from '../../../../hooks/useAppSelector'
+import { findMapFeatureById } from '../../../../utils/findMapFeatureById'
+import { OverlayPositionOnCentroid } from '../../../map/overlays/OverlayPositionOnCentroid'
+
+import type { BaseMapChildrenProps } from '../../../map/BaseMap'
+
+export function BaseOverlay({ currentFeatureOver: hoveredFeature, map }: BaseMapChildrenProps) {
+  const base = useAppSelector(state => state.base)
```

Comment:
```suggestion
  const selectedBaseFeatureId = useAppSelector(state => state.base.selectedBaseFeatureId)
```

Je propose de ne plus utiliser une écoute sur un reducer complet, c'est très lourd pour redux. Ici, on n'a que `selectedBaseFeatureId` ?